src/store/state/state-model.ts

```diff
@@ -113,7 +113,16 @@ export const stateModel: StateModelIF = {
   defineCompanyStep: {
     valid: false,
     cooperativeType: null,
-    officeAddresses: {}
+    officeAddresses: {
+      registeredOffice: {
+        mailingAddress: { ...EmptyAddress },
+        deliveryAddress: { ...EmptyAddress }
+      },
+      recordsOffice: {
+        mailingAddress: { ...EmptyAddress },
+        deliveryAddress: { ...EmptyAddress }
+      }
```

Comment:
Adding this and conditional logic change to set default address in 'RestorationBusinessInformation.vue' file resolved the issue with null address object issue